apps/api/src/app/events/e2e/trigger-event.e2e.ts

```diff
@@ -656,6 +657,19 @@ describe(`Trigger event - ${eventTriggerPath} (POST)`, function () {
 
       expect(deletedIntegrations.modifiedCount).to.eql(integrationIdsToDelete.length);
 
+      await integrationRepository.update(
+        {
+          _organizationId: session.organization._id,
+          _environmentId: session.environment._id,
+          active: false,
+        },
+        {
+          $set: {
+            active: true,
+          },
+        }
```

Comment:
I do not understand this addition. Why do you need to set all the integrations to active? In the creation of the organization Novu providers should be enabled, right? What I am missing?

Reply:
Because we handle everything from repository level so we need to activate Novu providers here to test them.